From 956e7d89865b497f889483c90bc7fed9bec4e030 Mon Sep 17 00:00:00 2001
From: Gyorgy Sarvari <skandigraun@gmail.com>
Date: Sun, 26 Oct 2025 18:55:06 +0100
Subject: [PATCH 1/2] fix typo in test script

While executing the test suite from tests/shell/testcases folder, a lot of errors
are displayed:

tests/shell/testcases/maps/vmap_timeout: line 48: [: : integer expected

Looking at the script, a non-existing variable (expires) is tested instead of
the existing one (expire).

This change corrects this.

Upstream-Status: Submitted [https://lore.kernel.org/netfilter-devel/20251026204107.2438565-1-skandigraun@gmail.com/T/#u]

Signed-off-by: Gyorgy Sarvari <skandigraun@gmail.com>
---
 tests/shell/testcases/maps/vmap_timeout | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/shell/testcases/maps/vmap_timeout b/tests/shell/testcases/maps/vmap_timeout
index 8ac7e8e7..55d1c1b9 100755
--- a/tests/shell/testcases/maps/vmap_timeout
+++ b/tests/shell/testcases/maps/vmap_timeout
@@ -45,7 +45,7 @@ for i in $(seq 1 100) ; do
 		expire=$((RANDOM%utimeout))
 
 		expire_str=""
-		if [ "$expires" -gt 0 ]; then
+		if [ "$expire" -gt 0 ]; then
 			expire_str="expires ${expire}s"
 		fi
 
